function.py

```diff
@@ -0,0 +1,6 @@
+
+def function(n):
+    if n == 0:
```

Comment:
For our recursion function, can we ensure that all base cases are covered?
```suggestion
    if n == 0 or n==1:
```